### DISCUSSION TITLE
Add specific time for color change and no transition between colors

### DISCUSSION THREAD

**Discussion by bit4us** [vydimitrov/react-countdown-circle-timer#181] [Ideas]
Hi,
Please let me tell you that I love your work. 
However for a project I'm working on, I'd be needing the possibility to set exact time when the color will change (without complicated calculations) and to have the option for no transition during color change (instant switch?). 
More exactly, I'd like to be able to set timer colors like this: 
duration={150}
colors={[["#42b0f5", 60],["#2bb32b",30], ["#eeee60", 30], ["#ff4a2e"]]}

or

duration={1350}
colors={[["#42b0f5", 1080],["#2bb32b",120], ["#eeee60", 120], ["#ff4a2e"]]}

As you can see in the examples I need to have the possibility to put in seconds instead of fraction when the color changes. 

Do you think you can make it, or is it already available?

**Comment by vydimitrov** [comment]
Hey @bit4us, you can achieve this with the current API and I will consider adding it as an option with the next major release, which I am planning to start soon. Check this [demo](https://codesandbox.io/s/ecstatic-driscoll-q1v75?file=/src/index.js) to see how you can implement it without transition. Regarding the time in seconds for now you will need to divide the color time by the duration, and pass it as a fraction, like so:
```js
duration={150}
colors={[["#42b0f5", 60/150],["#2bb32b",30/150], ["#eeee60", 30/150], ["#ff4a2e"]]}
```

**Reply by vydimitrov** [reply]
Added this feature request - https://github.com/vydimitrov/react-countdown-circle-timer/discussions/67#discussioncomment-1562707

**Comment by bit4us** [comment]
Hi,

Thank you for prompt response. Yes, the demo you’ve sent me helped me a lot. It’s great.

Thank you again and best regards.

Marius
